web/api/v0/__init__.py

```diff
@@ -77,19 +76,14 @@ def get_interface_or_404(interface_id):
 
 
 def generate_user_data(user):
-    try:
-        traffic_maxmium = effective_traffic_group(user).credit_limit
-    except NoTrafficGroup:
-        traffic_maxmium = None
-
     props = {prop.property_name for prop in user.current_properties}
     user_status = status(user)
 
     interval = timedelta(days=7)
     step = timedelta(days=1)
     traffic_history = func_traffic_history(user.id,
                                           session.utcnow() - interval + step,
-                                          interval, step)
```

Comment:
Looks broken to me, as the new traffic_history function expects `(start_date, end_date)` instead of `(start_date, interval, step)`.

Reply:
Whoever created the API should have written tests for it!

Reply:
Fixed.